GLib-2.0.gir

```diff
@@ -17356,6 +17380,16 @@ the pointer passed to it should not be `volatile`.</doc>
           </parameter>
         </parameters>
       </function>
+      <function name="init_enter_pointer" c:identifier="g_once_init_enter_pointer">
```

Comment:
Needs update of gobject-introspection

Reply:
Also is it expected that the GNOME SDK is already following next GLib? Asking because graphene looks like it was downgraded to a release.

Reply:
> Also is it expected that the GNOME SDK is already following next GLib

Yes, it tracks glib from main

> Asking because graphene looks like it was downgraded to a release.

We switched from tracking `master` to tracking tags for graphene, so it went from `1.10.6-42-g0cfa05ff62f244e4d5e7ac35a1979a23f25c5151`  to ` 1.10.8-0-g4e2578450809c2099400cf85caf18eafcd7100aa`

Reply:
Ok, so this is all expected and the only thing we need is to get gobject-introspection updated for the GLib introspection data.

Reply:
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/423